src/services/datahub-client.ts

```diff
@@ -153,7 +153,7 @@ export class HttpDatahubClient implements DatahubClient {
 
       // By the moment we are hard-coding it because DataHub does not have this
       // information. I am looking you Leo :P
-      const urlSchema = 'http://';
+      const urlSchema = response.data.protocol ? response.data.protocol : 'http://';
```

Comment:
I prefer this syntax, but it is the same:

```suggestion
      const urlSchema = response.data.protocol || 'http://';
```